controller/MiNiFiController.cpp

```diff
@@ -178,7 +177,7 @@ int main(int argc, char **argv) {
       auto& components = result["stop"].as<std::vector<std::string>>();
       for (const auto& component : components) {
         auto socket = secure_context != nullptr ? stream_factory_->createSecureSocket(host, port, secure_context) : stream_factory_->createSocket(host, port);
-        if (!stopComponent(std::move(socket), component))
+        if (stopComponent(std::move(socket), component))
```

Comment:
I agree this is definitely the right intent, but it appears as though we are not handling/propagating the returned value for writeData in sendSingleCommand which always returns true.

Reply:
The intent was to actually return false when writes failed. This is an oversight. Thanks